SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -82,6 +82,8 @@ SCDocHTMLRenderer {
 			};
 		};
 
+		if(thisProcess.platform.name == \windows) {result = result.replace("\\", "/")}; // replace backward with forward slash on Windows
```

Comment:
Couldn't we make this somehow more universal via

```
// enforce unix path for html
result = result.replace(Platform.pathSeparator, "/");
```

Platform dependent if statements can lead to some trouble while testing